setup.cfg

```diff
@@ -64,7 +64,7 @@ install_requires =
     schema
     simple-di>=0.1.4
     starlette
-    typing-extensions>=4.0
+    typing-extensions>=3.7,<4
```

Comment:
Why do we set an upper lock version for typing-extension?

Reply:
read the doc

Reply:
no need to lock it

Reply:
I think we have gone through this b4.